### PR TITLE
Change color of top bar in summary pop up

### PR DESCRIPTION
**Pull request by janKozie1**
for: openstax/unified#1228

### CODE REVIEW

File: src/app/content/styles/PopupStyles.tsx [hunk 1]
```diff
@@ -18,6 +20,17 @@ export const popupBodyPadding = 2.4;
...
 export const topBottomMargin = headerHeight + popupBodyPadding;
 
+const swapColors = ({colorSchema}: {colorSchema: BookWithOSWebData['theme']}) => {
+  const color = theme.color.primary[colorSchema].foreground;
+  return css`
+    color: ${color};
+
+    &:hover {
+      color: ${color === theme.color.text.white ? toolbarIconColor.base : toolbarIconColor.lighter};
+    }
```
Comment (PiotrKozlowski) [inline src/app/content/styles/PopupStyles.tsx:30]: This is how it looks when hovered for Anatomy and Physiology:
image

File: src/app/content/highlights/components/HighlightsPopUp.spec.tsx [hunk 2]
```diff
@@ -221,4 +225,35 @@ describe('MyHighlights button and PopUp', () => {
...
+
+    expect(component.toJSON()).toMatchSnapshot();
+
+    renderer.act(() => {
+      store.dispatch(closeMyHighlights());
+      store.dispatch(receiveBook({...book, theme: 'yellow'}));
+      store.dispatch(openMyHighlights());
+    });
+
+    expect(component.toJSON()).toMatchSnapshot();
```
Comment (PiotrKozlowski) [inline src/app/content/highlights/components/HighlightsPopUp.spec.tsx:257]: This theme change is not really visible in snapshots, right? Maybe just check for props on heading and close icon?

**Review by PiotrKozlowski — CHANGES_REQUESTED** [review]

File: src/app/content/components/BookBanner.tsx [hunk 3]
```diff
@@ -55,6 +51,7 @@ export interface PropTypes {
   pageNode?: ArchiveTreeSection;
   book?: Book;
   hasUnsavedHighlight?: boolean;
+  bookTheme?: BookWithOSWebData['theme'];
```
[the remark this answers is not in the document]
Comment (janKozie1) [reply src/app/content/components/BookBanner.tsx]: I wanted to keep the first test prop-less but this is probably better

File: src/app/content/highlights/components/HighlightsPopUp.spec.tsx (hunk 2 above)
Comment (janKozie1) [reply src/app/content/highlights/components/HighlightsPopUp.spec.tsx]: Only the class name changes. I tried testing for background with `getComputedStyle` but it doesn't seem to contain information about it.

**Review by PiotrKozlowski — APPROVED** [review]

**Pull request merged**